R/pkg/R/generics.R

```diff
@@ -623,6 +623,10 @@ setGeneric("getItem", function(x, ...) { standardGeneric("getItem") })
 
 #' @rdname column
 #' @export
+setGeneric("isNaN", function(x) { standardGeneric("isNaN") })
```

Comment:
sorry, rereading the description. it sounds like we have
`isNaN` for Column
`isnan` for DataFrame
?
That seems a bit confusing..